flow/table.go

```diff
@@ -170,3 +170,33 @@ func (ft *FlowTable) FilterLast(last time.Duration) []*Flow {
 	ft.lock.RUnlock()
 	return flows
 }
+
+func (ft *FlowTable) SelectLayer(endpointType FlowEndpointType, list []string) []*Flow {
+	meth := make(map[string][]*Flow)
+	ft.lock.RLock()
+	for _, f := range ft.table {
+		layerFlow := f.GetStatistics().Endpoints[endpointType.Value()]
+		if layerFlow.AB.Value == "ff:ff:ff:ff:ff:ff" || layerFlow.BA.Value == "ff:ff:ff:ff:ff:ff" {
+			continue
+		}
+		meth[layerFlow.AB.Value] = append(meth[layerFlow.AB.Value], f)
+		meth[layerFlow.BA.Value] = append(meth[layerFlow.BA.Value], f)
+	}
+	ft.lock.RUnlock()
+
+	mflows := make(map[*Flow]struct{})
```

Comment:
Yeah I known we can merge the uniq() with the flow list append, but it's less readable
I don't known I still divided